<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="UserGroupSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User and Group Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>&amp;Users</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTreeWidget" name="userList">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>Login Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>User ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Full Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Group</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Home Directory</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showSystemUsers">
          <property name="text">
           <string>Show system users (for advanced users only)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>&amp;Groups</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeWidget" name="groupList">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Group ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Members</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionDelete"/>
   <addaction name="actionProperties"/>
   <addaction name="actionChangePasswd"/>
   <addaction name="actionRefresh"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd">
   <property name="icon">
    <iconset theme="list-add">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add new users or groups</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="list-remove">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete selected item</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="icon">
    <iconset theme="document-properties">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Properties</string>
   </property>
   <property name="toolTip">
    <string>Edit properties of the selected item</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh the lists</string>
   </property>
  </action>
  <action name="actionChangePasswd">
   <property name="icon">
    <iconset theme="dialog-password">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
   <property name="toolTip">
    <string>Change password for the selected user or group</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
